frontend/src/helpers/dom/format-date/format-date.ts

```diff
@@ -0,0 +1,12 @@
+import { FormatDate } from 'common/types/types';
```

Comment:
move to `frontend/src/helpers/date/get-formatted-date/get-formatted-date.helper.ts`